src/libecalc/fixtures/cases/all_energy_usage_models/data/all_energy_usage_models.yaml

```diff
@@ -184,6 +184,8 @@ MODELS:
           CONTROL_MARGIN_UNIT: PERCENTAGE # PERCENTAGE is default, FRACTION is other possible value
         - INLET_TEMPERATURE: 30
           COMPRESSOR_CHART: predefined_compressor_chart
+          CONTROL_MARGIN: 0
+          CONTROL_MARGIN_UNIT: PERCENTAGE # PERCENTAGE is default, FRACTION is other possible value
```

Comment:
Should the `CONTROL_MARGIN_UNIT` be moved to where the other units (`RATE`, `HEAD`, `EFFICIENCY`) are defined for a compressor chart? It may complicate things it will be required for single and variable speed trains, but not for simplified trains (simplified uses generic charts, that are variable speed charts built on input).

Reply:
It seems logical to do that. Maybe a separate issue/PR?

Reply:
Yes, sure. Just make an issue so we don't forget.

Reply:
Shouldn't `CONTROL_MARGIN` also be moved to compressor_chart in that case?